### ISSUE TITLE
Publish on NPM registry

### ISSUE THREAD

**Issue by matheussampaio** [bonzaiferroni/Traveler#13]
Do you have plans to publish this library on `npmjs` registry?

**Comment by bonzaiferroni** [comment]
Yes, I'd like to publish traveler and bonzAI on npm, I just haven't had a chance to figure out what I need to do.

**Comment by matheussampaio** [comment]
You can find [here](https://docs.npmjs.com/getting-started/publishing-npm-packages). Basically, you need to create a user, find an unique name for the package (Traveler is already taken) and run npm publish every time you want to publish a new version.

**Comment by bonzaiferroni** [comment]
Thanks!
